terminado/tests/test_basic.py

```diff
@@ -0,0 +1,46 @@
+#
+# basic_tests.py -- Basic unit tests for Terminado
+#
+
+from __future__ import absolute_import, print_function
+
+import unittest
+from terminado import SingleTermManager, TermSocket
+import tornado
+import tornado.httpserver
+import tornado.httpclient
+import tornado.testing
+import logging
+import json
+
+class BasicTest(tornado.testing.AsyncHTTPTestCase):
+	def get_app(self):
+		return tornado.web.Application([
+					(r"/websocket/(\w+)", TermSocket,
+						{'term_manager': SingleTermManager(shell_command=['bash'], ioloop=self.io_loop)}),

```

Comment:
It looks like you're indenting with tabs, which Github is rendering as 8 spaces, sending lines off to the right. Can you use 4-space indentation for consistency with the regular terminado code?